demisto_sdk/commands/format/update_readme.py

```diff
@@ -0,0 +1,135 @@
+                                   re.IGNORECASE | re.MULTILINE)
+        relative_urls += re.findall(RELATIVE_HREF_URL_REGEX, self.readme_content,
+                                    re.IGNORECASE | re.MULTILINE)
+        relative_urls = [url for url in relative_urls if url[1]]
+        return relative_urls
+
+    def get_new_url_from_user(self, url: list) -> Optional[str]:
+        """ Given we found a relative url, the user has the following options-
+        1. Add https:// prefix.
+        2. Enter a new absolute address to replace the current.
+        3. Leave as is
+
+        Args:
+            url: relative url found in README
+
+        Returns: new url or None if we leave as is
+
+        """
+        old_url = str.strip(url[1])
+        new_address = None
+        if self.assume_yes:
+            new_address = 'https://' + old_url
+        else:
+            click.secho(f'Relative urls are not supported within README, Should https:// be added to the '
+                        f'following address? [Y/n]\n {url[1]}',
+                        fg='red')
+            user_answer = input()
+            if user_answer in ['y', 'Y', 'yes', 'Yes']:
+                new_address = 'https://' + old_url
+            else:
+                click.secho('Would you like to change the relative address to something else?\n'
+                            ' Enter the new address or leave empty to skip:',
+                            fg='red')
+                user_answer = input()
+                if user_answer and user_answer not in ['n', 'N', 'No', 'no']:
+                    new_address = user_answer
+        return new_address
+
+    def relative_url_format(self):
+        """Find all relative url links in README,
+            And prompt the user for following options-
+        1. Add https:// prefix.
+        2. Enter a new absolute address to replace the current.
+        3. Leave as is
+        """
+
+        relative_urls = self.get_relative_urls()
+
+        for url in relative_urls:
+            new_address = self.get_new_url_from_user(url)
+            if new_address:
+                self.replace_url_in_content(url, new_address)
+
+    def save_md_to_destination_file(self):
+        """Safely saves formatted data to destination file."""
+        if self.source_file != self.output_file and self.verbose:
+            click.secho(f'Saving output description file to {self.output_file} \n', fg='white')
+        with open(self.output_file, 'w') as f:
+            f.write(self.readme_content)
+        f.close()
+
+    def run_format(self) -> int:
+        try:
+            click.secho(f'\n================= Updating file {self.source_file} ================= ', fg='bright_blue')
+            self.relative_url_format()
+            self.save_md_to_destination_file()
+            return SUCCESS_RETURN_CODE
+        except Exception as err:
+            if self.verbose:
```

Comment:
why only if verbose? should errors not be printed otherwise?

Reply:
It's based on other format classes, removed the verbose condition..

Reply:
I think it should be printed